map/manage-operational-layers/src/main/java/com/esri/samples/manage_operational_layers/ManageOperationalLayersSample.java

```diff
@@ -38,131 +38,156 @@
 import com.esri.arcgisruntime.mapping.ArcGISMap;
 import com.esri.arcgisruntime.mapping.Basemap;
 import com.esri.arcgisruntime.mapping.LayerList;
+import com.esri.arcgisruntime.mapping.Viewpoint;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
 public class ManageOperationalLayersSample extends Application {
 
   private MapView mapView;
-  private LayerList mapAddedLayers;
+  private LayerList mapLayers;
 
-  private static final String ELEVATION_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer";
   private static final String CENSUS_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer";
+    "https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer";
   private static final String DAMAGE_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer";
+    "https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer";
```

Comment:
When testing the sample, I can't see this layer having any content. I had a look at the service, and it doesn't look like there are actually any interesting features in it:

![Snag_2bd09d](https://user-images.githubusercontent.com/44193148/95175206-accec980-07aa-11eb-844e-c4a65d367688.png)

https://www.arcgis.com/home/webmap/viewer.html?featurecollection=https%3A%2F%2Fsampleserver5.arcgisonline.com%2Farcgis%2Frest%2Fservices%2FDamageAssessment%2FMapServer%3Ff%3Djson%26option%3Dfootprints&supportsProjection=true&supportsJSONP=true

Might be worth mentioning to the team and seeing if there's more interesting data to put in there, so it's visually clearer. Unless there was a thought behind there being no data in one of the layers?

Reply:
![image](https://user-images.githubusercontent.com/56556822/95192226-caf7f200-07c9-11eb-9c37-e23c278b9e72.png)

@JonLavi I might be misunderstanding but the damage assessment has some data when you zoom out, so gives another layer to play with with the controls?

Reply:
I would be in favour of having a more zoomed out starting point for this sample: it's not instantly clear you have to zoom out to see the damage assessment layer and like Jon says, it doesn't look too interesting. 

If the map was scaled to `10000000` in the constructor, you could see a few points: however, these points will move around over time so I'd be more in favour of centralising the viewpoint mid-North America (pretty much so it loads looking like the screenshot you have above: though if you do that, you might want to disable the UI until the world elevation layer has finished loading since it might be a little slow).

Reply:
or we could manually add lots of points in around Redlands ;)

Reply:
My bad, I didn't realise you need to zoom out that far... I don't mind either way, but probably sensible to have some data there that shows at the initial viewpoint that we set in the code

Reply:
I've updated the initial viewpoint, think it works way better. As the user you're keen to use the functionality and get that damage layer to the top. Thanks both for the discussion! @Rachael-E @JonLavi 

![image](https://user-images.githubusercontent.com/56556822/95223063-11ad1280-07f1-11eb-9507-4856fdac50f0.png)